src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java

```diff
@@ -132,7 +132,8 @@ private ReleaseCreationInformation searchEventInFollowingBlocks(
         Keccak256 informingRskTxHash
     ) throws HSMReleaseCreationInformationException {
         Block block = blockStore.getChainBlockByNumber(blockNumber);
-        // If the block cannot be found by its number, the event cannot be searched further.
+        // If the block cannot be found by its number, the event cannot be
+        // searched further.
```

Comment:
Why adding this second line?

Reply:
I tend to see an 80 column limit in most repos. Looking at the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit), I noticed it is recommended to be 100. Let me update that. Btw, maybe it would be worth it to set auto-formatting using [google-java-format](https://github.com/google/google-java-format) fmt, wdyt?

Reply:
Thanks for clarifying. I was using 120(Intellij default). Since yesterday, I added google-java-style to Intellij, so now it is 100.